tests/messages.spec.js

```diff
@@ -225,6 +225,7 @@ describe('Messages', () => {
         .send(deschant)
         .end((err, res) => {
           testUsertoken = res.body.data.token;
+					console.log('TCL: testUsertoken', testUsertoken);
```

Comment:
Why this `console.log`?

Reply:
It is a forgotten console.log I used for debugging. Will remove it now.